opensrp-anc/src/main/java/org/smartregister/anc/library/repository/PatientRepository.java

```diff
@@ -156,8 +152,12 @@ public static void updateEDDDate(String baseEntityId, String edd) {
         } else {
             contentValues.putNull(DBConstantsUtils.KeyUtils.EDD);
         }
+        updatePatient(baseEntityId, contentValues, getRegisterQueryProvider().getDetailsTable());
+    }
+
+    public static void updatePatient(String baseEntityId, ContentValues contentValues, String detailsTable) {
```

Comment:
If `detailsTable` is always the same, do the calling functions really need to always specify it? 

i.e. could the previous `getRegisterQueryProvider().getDetailsTable()` still work?